common/common.go

```diff
@@ -669,15 +669,15 @@ func SortStrings[S ~[]E, E fmt.Stringer](x S) S {
 
 // Counter is a thread-safe counter.
 type Counter struct {
-	n int64 // privatised so you can't use counter as a value type
+	n atomic.Int64 // privatised so you can't use counter as a value type
```

Comment:
A wise man once said:

"This realignment due to the requirement for assignment alignment brings no excitement, only chastisement that my review indictment brought defilement"

Howver this change rids the need for such things 😆